Adyen/UI/Form/Items/Text/FormTextItemView.swift

```diff
@@ -286,6 +282,14 @@ open class FormTextItemView<ItemType: FormTextItem>: FormValueItemView<String, F
             titleLabel.textColor = defaultTitleColor
         }
     }
+    
+    /// :nodoc:
+    public func notifyDelegateOfMaxLengthIfNeeded() {
+        let maximumLength = item.validator?.maximumLength(for: item.value) ?? .max
+        if item.value.count == maximumLength {
+            delegate?.didReachMaximumLength(in: self)
+        }
+    }
```

Comment:
nice 👍

Reply:
maybe it should be `if item.value.count >= maximumLength`, since by the time bin lookup response comes back the shopper maybe have already entered a number greater than max length, right?

Reply:
Thats a good catch, let me try both cases.